.docheader

```diff
@@ -0,0 +1,7 @@
+/*
+ * This file is part of the prooph/service-bus.
+ * (c) 2014-%year% prooph software GmbH <contact@prooph.de>
```

Comment:
add here another line:

    (c) 2014-%year% Sascha-Oliver Prolic <saschaprolic@googlemail.com>

It was on our todo list for a long time to update to license headers :)